getting-started/scaling-out.md

```diff
@@ -0,0 +1,144 @@
+# Scaling out using Distributed Hypertables
+
+TimescaleDB supports multi-node clustering by leveraging the hypertable and chunk primitives.
+Hypertables are used to handle large amount of data by breaking it up
+in smaller pieces (chunks), allowing operations to execute efficiently. When
+the amount of data is expected to be beyond what a single machine can handle,
+you can distribute the data chunks over several machines by using
+*distributed hypertables*.
+Distributed hypertables are similar to normal hypertables, but they
+add an additional layer of hypertable partitioning by distributing chunks
+across *data nodes*.
+
+In the multi-node topology ([architecture][]), all nodes are TimescaleDB instances.
+The data are distributed and stored in TimescaleDB instances, called data nodes.
+The distributed data are accessed through distributed hypertable
+on another TimescaleDB instance, called the *access node*.
+The access node is the entry point for any access to data in the cluster.
+
+## Working with Data Nodes
+
+Data nodes act as the containers for the hypertable chunks and are
+necessary to create distributed hypertables. Data nodes are
+added to the current database using [`add_data_node`][add_data_node]
+and removed using [`delete_data_node`][delete_data_node].
+
+Note that:
+
+* Data nodes are added as objects locally and just connect to an
+  existing database server.
+
+* You should already have a running database server on some host.
+
+When creating the data node, you should:
+
+* Provide a (local) name to use when referring to the data node from
+  this database.
+
+* Provide the host where the hypertable partition for the distributed
+  hypertable should be stored.
+
+* Ensure that the user used for connecting to the data node has
+  `CREATEDB` permissions or be a superuser. This is necessary since
+  `create_distributed_hypertable` expect to be able to create a
+  database on the remote data node.
```

Comment:
Alright, there's a few things wrong with this paragraph, although I'm not sure how much depth we want to go into here given how complicated this whole thing is.  First I'd be more explicit that the user used for connecting is the bootstrap user, as the current user doesn't need elevated permissions on the target postgres server (they'll be set as the owner of the data node target database, which is all the permission they need).  `CREATEDB` probably isn't sufficient for the bootstrap user either, as that may not be enough to load the timescale extension.  `create_distributed_hypertable` also doesn't create new databases, I assume this is supposed to be `add_data_node`.

Reply:
I improved formulation with small fixes. `CREATEDB` privilege is enough, since create extension requires the database owner, which I expect the bootstrap user is going to be after `add_data_node` created the database.

Reply:
The bootstrap user actually creates the database with the local user as the owner.

Reply:
According to [Postgres docs](https://www.postgresql.org/docs/11/sql-createextension.html) the user, who creates extension, should be superuser or database owner. More specifically the user should have all necessary permissions to execute all operations needed by the extension to be loaded. Should we use superuser for now?

Reply:
Actually, let's leave the documentation the way it is for now even if it's quite correct for the current code.  I'll change the bootstrapping to actually use the current user (who was made the database owner) to load the extension.  This has the added benefit of validating that we can connect the user with the provided password, which is something we're currently missing.

Reply:
I have already changed that superuser is required for the bootstrap user. It is in sync with suggestion from @erimatnor in `add_data_node` doc.